guide-us-english.tex

```diff
@@ -199,73 +199,51 @@ \subsection{Team-Level Process}
 the team level process are to:
 \begin{itemize}
 \item maximize the flow of completed and quality tested work.
-\item increase velocity a little each Sprint.
+\item increase performance of the team over time.
 \item operate in a way that is sustainable and enriching for the team.
 \end{itemize}
 
 \subsection{Coordinating the ``How'' - The Scrum of Scrums}
-A set of the teams that have a need to coordinate comprise a
-\textbf{``Scrum of Scrums'' (SoS)}. The SoS is a ``team of
-teams,''\footnote{McChrystal, General Stanley and Collins, Tantum and
-Silverman, David and Fussell, Chris, Team of teams: New rules of engagement
-for a complex world, Penguin, 2015} which hold a \textbf{Scaled Daily Scrum
-(SDS)} event with a representative from each team (usually the team's Scrum
-Master, although any person or number of people may attend). The SDS exists
-to coordinate teams and remove impediments to delivering value.
-
-The SDS event mirrors the Daily Scrum in that it optimizes the
-collaboration and performance of the network of teams. Additionally, the
-SDS:
+A set of the teams that have a need to coordinate comprise a \textbf{``Scrum of Scrums'' (SoS)}. This team is a Scrum Team unto itself which is responsible for a fully integrated set of potentially shippable increments of product at the end of every Sprint from all participating teams. A SoS functions as a Release Team and must be able to directly deliver value to customers. To do so effectively, it needs to be consistent with the Scrum Guide; that is, have its own roles, artifacts, and events:
```

Comment:
We're not sure this addresses the core confusion that appears in the first three lines

A set of the teams that have a need to coordinate comprise a \textbf{``Scrum of Scrums'' (SoS)}. This team is a Scrum Team unto itself which is responsible for a fully integrated set of potentially shippable increments of product at the end of every Sprint from all participating teams. A SoS functions as a Release Team and must be able to directly deliver value to customers.

Sentence one says, "SoS = Set of teams"
Sentence 2-3 says, "SoS = a scrum tream of representatives from those teams"

Can we change the first sentence to say,

"A set of teams that need to coordinate do so through a \textbf{``Scrum of Scrums'' (SoS)}."  The SoS is a Scrum Team unto itself..."

Reply:
The Scrum of Scrums is the set of teams that delivers a product increment.
The Scaled Daily Scrum is a meeting of representatives of those teams and anyone else needed to deliver a shippable increment. Wording needs to be changed to make this consistent.